ga4gh/datamodel/__init__.py

```diff
@@ -381,18 +381,6 @@ def sanitizeAlignmentFileFetch(
         return referenceName, start, end
 
     @classmethod
-    def sanitizeFastaFileFetch(cls, start=None, end=None):
```

Comment:
why are we removing this?  it looks like we're still calling into fastaFile.fetch ...